app/views/layouts/_form_errors.html.erb

```diff
@@ -1,11 +1,11 @@
 <% if defined? resource %>
   <% if resource&.errors&.any? %>
-    <div class="govuk-error-summary">
+    <div class="govuk-error-summary" id="error-summary">
       <h2 class="govuk-error-summary__title">There is a problem</h2>
       <div class="govuk-error-summary__body">
         <ul class="govuk-list govuk-error-summary__list">
           <% resource.errors.full_messages.each do |message| %>
-            <li><%= message %></li>
+            <li><%= message.html_safe %></li>
```

Comment:
[Having a discussion about this](https://github.com/alphagov/govwifi-admin/pull/514#discussion_r263419156) on the other open PR!